deploy/development/config.yml

```diff
@@ -9,5 +9,5 @@ data:
   WP_SITEURL: 'https://dev.intranet.justice.gov.uk/wp'
   AWS_CLOUDFRONT_HOST: 'cdn.dev.intranet.justice.gov.uk'
   # See Azure Setup for more information on how to get these values.
-  OAUTH_CLIENT_ID: "1dac3cbf-91d2-4c0e-9c80-0bf3f8fabd75"
-  OAUTH_TENANT_ID: "c6874728-71e6-41fe-a9e1-2e8c36776ad8"
+  OAUTH_CLIENT_ID: "51266573-d4eb-41db-8ebe-a9548aa4f01e"
+  OAUTH_TENANT_ID: "0bb413d7-160d-4839-868a-f3d46537f6af"
```

Comment:
These might (will) get flagged by the security squad at some point. Do they need to be hidden or are we good to roll with them?

Reply:
Ass discussed, I'll add a concise note as to why these are ok here.